packages/example/src/components/pages/SliderFormWithSkipsInHooks.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-alert, react/prefer-stateless-function */
+import React, { Component } from 'react';
+import { Slide, Slider } from '@swan-form/slider';
+import { hot } from 'react-hot-loader';
+
+const onSubmit = values => {
+  alert(JSON.stringify(values));
```

Comment:
do we want an alert here?

Reply:
Yeah, that gives the user some feedback that they submitted.